deploy/eck-operator/values.yaml

```diff
@@ -165,6 +168,37 @@ config:
   # Can be disabled if cluster-wide storage class RBAC access is not available.
   validateStorageClass: true
 
+# Prometheus PodMonitor configuration
+# Reference: https://docs.openshift.com/container-platform/4.5/rest_api/monitoring_apis/podmonitor-monitoring-coreos-com-v1.html
```

Comment:
I think the reference is here: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#podmonitor, it would avoid the reference to Openshift 4.5.

Reply:
Updated to correct  reference, thanks for noticing